docs/use-the-network/coverage-mapping/mappers-quickstart.mdx

```diff
@@ -42,7 +42,7 @@ integration.
 
 Next we'll fill in the details, see instructions and image below.
 
-1. Enter the Mappers API Ingest endpoint
+1. Enter the Mappers API Endpoint URL (Required)
```

Comment:
How about the combination of these.. `Ingest Endpoint URL`

Reply:
@waveform06 Can we get this updated per Kent's suggestion?